client/client_test.go

```diff
@@ -1888,7 +1889,7 @@ func testOCILayoutPlatformSource(t *testing.T, sb integration.Sandbox) {
 			},
 		},
 		OCIStores: map[string]content.Store{
-			csID: store,
+			"oci:" + csID: store,
```

Comment:
I don't understand this change. The name of the store is chosen by the caller. It can be anything and can have a prefix if the client chooses it. We shouldn't arbitrarily change it. The name that caller passes in here and in LLB/Frontend-args should be identical.

If we want to make sure that these do not collide with other contentstores then this would need to be handled by the client library implementation. Eg. when calling `s.Allow(sessioncontent.NewAttachable(contentStores))` OCI stores are registered differently(either prefix or second type parameter next to the name).

Reply:
> The local cache uses local:, 

This isn't exposed to the user. We could really just use a random identifier in there.

For the "oci-layout" the value is actually important as the user needs to use the exact same value on setting `OCIStores` and on creating the LLB definition. Also the name user chooses in there should be anything. If buildkit internally namespaces it by type field or prefix, then that should be invisible to the user/client public API.

Reply:
Right! Understood, I'll add the prefixing here:

> Eg. when calling s.Allow(sessioncontent.NewAttachable(contentStores)) OCI stores are registered differently(either prefix or second type parameter next to the name).

Sorry, I understand your point now :tada: